src/components/WifiCard.js

```diff
@@ -38,7 +38,7 @@ export const WifiCard = (props) => {
       ? ''
       : escape(props.settings.password);
     setQrvalue(
-      `WIFI:T:${props.settings.encryptionMode};S:${ssid};P:${password};;`
+      `WIFI:T:${props.settings.encryptionMode};S:${ssid};P:${password};H:${props.settings.hiddenSSID}`
```

Comment:
Noting that this will change the QR Code data for a non-hidden network. Given a network `foo` and password `bar`, the previous output was:
```
WIFI:T:WPA;S:foo;P:bar;;
```
The trailing semicolon is now removed and replaced:
```
WIFI:T:WPA;S:foo;P:bar;H:false
```

I've tested this on my own non-hidden network and it works for me, but worth noting since these protocols may be picky across devices; hard to know when the documentation is so fragmented.

Reply:
I add semicolon on the end of this common format, so now the result is eg.:
`WIFI:T:WPA;S:foo;P:bar;H:false;`